src/Controllers/GagePageController.ts

```diff
@@ -190,6 +190,10 @@ module StreamStats.Controllers {
         public peakValuePlot: any;
         public annualFlowPlot: any;
         public peakValues: any;
+        public dischargeObj = undefined; // Stage vs. Discharge Plot
+        public ratingCurve = undefined; // Stage vs. Discharge Plot
```

Comment:
This variable only seems to be used in commented out code. If you don't plan on using it in the future it would be good to clean up along with the related comments.